README.md

```diff
@@ -1,31 +1,53 @@
 # tket
 
+[![Slack](https://img.shields.io/badge/Slack-4A154B?style=for-the-badge&logo=slack&logoColor=white)](https://tketusers.slack.com/join/shared_invite/zt-18qmsamj9-UqQFVdkRzxnXCcKtcarLRA#/shared-invite/email)
```

Comment:
Is the last part of this URL (`#/shared-invite/email`) necessary? Seems to work without it.

Reply:
I've removed this part. I don't think its needed.